inversify.config.ts

```diff
@@ -0,0 +1,108 @@
+import { Container } from "inversify";
+import TYPES from "./ioc.types";
+import {
+    SendDataHandler
+} from "./source/MessageHandlers";
+
+import { S3, SQS } from 'aws-sdk';
+import * as Winston from 'winston';
+import { AuthManager } from "./source/astra-sdk/AuthManager";
+import { CognitoUserPoolLocatorUserManagement } from "./source/astra-sdk/CognitoUserPoolLocatorUserManagement";
+import IMessageHandler from "./source/IMessageHandler";
+import { DiscoverySdk, BearerTokenCredentials } from "@adastradev/serverless-discovery-sdk";
+import { UserManagementApi } from "./source/astra-sdk/UserManagementApi";
+import MessageHandlerFactory from "./source/MessageHandlerFactory";
+import MessageFactory from "./source/MessageFactory";
+import SendDataMessage from "./source/Messages/SendDataMessage";
+import IMessage from "./source/IMessage";
+
+
+// TODO: Make configurable?
+const REGION = "us-east-1";
+const stage = "prod";
+
+var container = new Container();
+
+// TODO: Get region
+var s3Config: S3.ClientConfiguration = { region: REGION };
+var sqsConfig: SQS.ClientConfiguration = { apiVersion: '2012-11-05', region: REGION };
+
+var logger: Winston.Logger = Winston.createLogger({
+    level: 'info',
+    format: Winston.format.json(),
+    transports: [
+        new Winston.transports.Console()
+    ]
+});
+process.env['DISCOVERY_SERVICE'] = 'https://4w35qhpotd.execute-api.us-east-1.amazonaws.com/prod';
+const sdk: DiscoverySdk = new DiscoverySdk(process.env.DISCOVERY_SERVICE, REGION);
+
+var poolLocator = new CognitoUserPoolLocatorUserManagement(REGION);
+var authManager = new AuthManager(poolLocator, REGION);
+var queueUrl: string, 
+    tenantId: string,
+    s3: S3,
+    sqs: SQS;
+
+let s3Buckets = {
+    prod: 'adastra-prod-data-ingestion',
+    dev: 'adastra-dev-data-ingestion'
+}
+let bucketName = s3Buckets[stage];
+
+// NOTE: updates to the discovery service itself would require pushing a new docker image. This should still be an environment variable rather than hardcoded
+process.env['DISCOVERY_SERVICE'] = 'https://4w35qhpotd.execute-api.us-east-1.amazonaws.com/prod';
```

Comment:
Up until this point the inversify config is really a startup script of sorts. Beyond this point it's preparing common instances that we'll likely need across the app as it runs like client objects, auth manager stuff and various other things.